### DISCUSSION TITLE
ll.syncs update for syncing to external midi clocks

### DISCUSSION THREAD

**Discussion by chausch** [ppooll-dev/ppooll#145] [General]
hi there,
i added an option to ll.syncs to sync to an external midi clock in the commit 3b17a92dfef3ee66ed8a4f9713c1b1d56e9e9f51
for this to work you have to set the "@ext_midi 1" patcher argument.
the option ext_midi then shows up in the dropdown list.
in order to use it you have to add a second umenu to your patch that is called "syncto_extmidi". this will be auto populated when  ext_midi is selected in the syncto menu.
this also enables the output of midi start/stop/tick messages via a new, fourth outlet.

o/t
i also fixed the sync out behaviour to reflect the current state of the patch. ie. when a patch is set to sync_in it should also transmit the input frequency. else if someone syncs an act A to another act B that is itself driven by a third sync C, A should sync to C as well.
sounds good?
all the best

**Comment by jpsteccato** [comment]
looking forward to trying this :)
